lib/gscraper_web/views/layout_view.ex

```diff
@@ -1,6 +1,8 @@
 defmodule GscraperWeb.LayoutView do
   use GscraperWeb, :view
 
+  alias GscraperWeb.SharedView
```

Comment:
I think we can remove this aliasing

Reply:
this alias was defined here as it was used in the layout template:

https://github.com/longnd/elixir-gscraper/blob/dcdd8cf7cb24a059f4bc4cf609633c7526df361c/lib/gscraper_web/templates/layout/app.html.eex#L12-L13

without that, we need to use a wordy namespace like this:
```elixir
<%= render GscraperWeb.SharedView, "_header.html", assigns %> 
```

or do you think it isn't worth do that?

Reply:
Got it, I'm fine with both approaches 👍